### ISSUE TITLE
unable to send custom emoji

### ISSUE THREAD

**Issue by DMJWorld** [telegramdesktop/tdesktop#28377]
### Steps to reproduce

send custom emoji or copy and past a caption with custom emoji

### Expected behaviour

to see custom emoji

### Actual behaviour

emoji standard

### Operating system

windows 10

### Version of Telegram Desktop

v.5.5.2

### Installation source

Static binary from official website

**Comment by ilya-fedin** [comment]
You need premium for that

**Comment by webportnoy** [comment]
Same bug on Linux after last update

**Comment by CarlosLopezES** [comment]
Same problem here, with MacOS and Windows 11.

**Comment by john-preston** [comment]
Sorry, I'll deploy a fix version later today.

**Comment by Kamenny** [comment]
Yes, I confirm, I also have such a problem since this morning, please fix it

**Comment by Andrey-Popov-Inside** [comment]
Same problem. Premium emotes are not sent as expected. Windows 10

**Comment by Aokromes** [comment]
no need of report anymore, preston said fix is comming.

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/releases/tag/v5.5.3
